raster_api/infrastructure/construct.py

```diff
@@ -69,6 +69,10 @@ def __init__(
             "VEDA_RASTER_PGSTAC_SECRET_ARN", database.pgstac.secret.secret_full_arn
         )
 
+        veda_raster_function.add_environment(
```

Comment:
add_environment() expects a string so we should conditionally add the path_prefix setting only if it is defined. 

Same change in the stac api construct

Reply:
Although if you instead go with the path_prefix default value is an empty string this would work without the condition.